source/tutorial/autocomplete-component.md

```diff
@@ -250,15 +250,15 @@ In the `filterByCity` function in the rentals controller above,
 we've added a new property called `query` to the filter results instead of just returning an array of rentals as before.
 
 ```app/components/list-filter.js{-19,-9,+10,+11,+12,+20,+21,+22,+23,+24}
-import Ember from 'ember';
+import Component from '@ember/component';
 
 export default Component.extend({
   classNames: ['list-filter'],
   value: '',
 
   init() {
     this._super(...arguments);
-    this.get("filter")("").then((results) => this.set("results", results))
+    this.get('filter')('').then((results) => this.set('results', results));
```

Comment:
In the example code avobe, it uses `'` and `;`.
I know it's kind of nitpicking but some people are nervous these things.